linalg/simd/sve.hpp

```diff
@@ -0,0 +1,287 @@
+// Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#ifndef MFEM_SIMD_SVE_HPP
+#define MFEM_SIMD_SVE_HPP
+
+#if defined(__aarch64__) && defined(__ARM_FEATURE_SVE)
+
+#include "../../config/tconfig.hpp"
+#include <arm_sve.h>
+#define PG svptrue_b64()
```

Comment:
`PG` is a very generic name for a macro. At least add an `MFEM_` prefix; consider expanding `PG` as well. What does it stand for?

Reply:
It's for the predicate group - or mask. I'll expand it, that will be easier.

Reply:
`done`